### PR TITLE
223 admin panel change user picture

### PR DESCRIPTION
**Pull request by hafizhanindito**
#223 

### Breakdown:
- [x] Add feature to admin panel to change picture

### CODE REVIEW

File: app/routes/main.py [hunk 1]
```diff
@@ -69,6 +70,11 @@ def get_stages():
 def schedules():
     return MainController.getSchedules()
 
+<<<<<<< HEAD
```
Comment (andy-shi88) [inline app/routes/main.py]: conflicts sssss

File: app/routes/main.py [hunk 2]
```diff
@@ -91,3 +97,4 @@ def sponsors():
 @main.route('/password')
 def changepassword():
     return MainController.changepassword()
+>>>>>>> develop
```
Comment (andy-shi88) [inline app/routes/main.py]: conflictssss

File: app/templates/admin/users/user_photos_add.html [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+<script>
+    
+    $('.btn-save').click(function(event){
+        var form = $('#upload-photo')[0];
+        var payload = new FormData();
+        payload.append('image_data', $('#i-user-photo')[0].files[0]);
+                   
+        dsa.postForm('user/photo', payload, function(result) {
+            if(result.meta.success) {
+                window.location = "http://localhost:5000/accounts";
```
Comment (andy-shi88) [inline app/templates/admin/users/user_photos_add.html]: just use `/accounts`

**Comment by hafizhanindito** [comment]
ok, the mess should be fixed

**Pull request merged**